metabase/metabase
# Mblib drill thru fe

Closes https://github.com/metabase/metabase/issues/31007

### Description

This adds usage of MLv2 for drills:
- FKFilterDrill
- SummarizeColumnByTimeDrill

For other implemented drills we have added unit tests and found some issues. We will cover them in upcoming PRs

### How to verify

Try to create different questions and try various drills.
Everything should work similarly to production.

### Checklist
